charts/s3gw/templates/configmap.yaml

```diff
@@ -7,9 +7,17 @@ metadata:
   labels:
 {{ include "s3gw.labels" . | indent 4}}
 data:
+{{- if .Values.ingress.enabled }}
+  RGW_DNS_NAME: >-
+    {{ .Values.serviceName }}.{{ .Values.publicDomain }},
+    {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
+{{- else}}
+  RGW_DNS_NAME: >-
+    {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
+{{- end }}
```

Comment:
Why inject this via configmap into the environment and then into the command line arguments from there and not directly into the command line arguments via the template that the helm chart provides?

Reply:
it is not clear to me what do you mean here; can you please elaborate more?
we are not using a configmap here for these values, or I'm not understanding properly the issue.

Reply:
ok, sorry, I realized now that `RGW_DNS_NAME` is a config map entry ;) 
Probably this comes from the early implementations we developed. We were not using helm charts, so probably this made sense at that time.
Can we maybe think to do some cleaning on a dedicated issue, what do you think?
I'd like not to do too much changes on a single PR.

Reply:
Fair. Open an issue, if you feel that this might get dropped, but lets not block the PR on this one if it works.